cfgov/unprocessed/js/molecules/Autocomplete.js

```diff
@@ -121,23 +130,60 @@ function Autocomplete( element, opts ) {
       inputCoords.left ) + 'px';
   }
 
+  /**
+   * Toggles the error state and message that's shown when the
+   * max search term length is hit
+   */
+  function _toggleMaxLengthError() {
+    _maxLengthExceeded = _searchTerm.length >= _settings.maxChars ? true :
```

Comment:
Should this test just be greater-than? Currently, the warning pops when a user *hits* 50 chars, effectively limiting search terms to 49 characters.

```suggestion
    _maxLengthExceeded = _searchTerm.length > _settings.maxChars ? true :
```